x-pack/plugins/upgrade_assistant/public/application/components/shared/search_bar/search_bar.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiButton,
+  EuiFieldSearch,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiCallOut,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { DomainDeprecationDetails } from 'kibana/public';
+import { DeprecationInfo } from '../../../../../common/types';
```

Comment:
Nit. We've recently gotten into the habbit of using `import type` when we are only importing types from modules. It avoides adding some unused javascript into the plugin bundle by mistake. Its also more declarative.

```suggestion
import type { DomainDeprecationDetails } from 'kibana/public';
import type { DeprecationInfo } from '../../../../../common/types';
```